Source/WebCore/Modules/websockets/WebSocketDeflater.cpp

```diff
@@ -180,18 +180,17 @@ bool WebSocketInflater::addBytes(const uint8_t* data, size_t length)
             return false;
         ASSERT(remainingLength > m_stream->avail_in);
     }
-    ASSERT(consumedSoFar == length);
+    ASSERT(consumedSoFar == data.size());
     return true;
 }
 
 bool WebSocketInflater::finish()
 {
-    constexpr uint8_t strippedFields[] = { 0, 0, 0xFF, 0xFF };
-    constexpr auto strippedLength = std::size(strippedFields);
+    constexpr std::array<uint8_t, 4> strippedFields { 0, 0, 0xFF, 0xFF };
 
     // Appends 4 octests of 0x00 0x00 0xff 0xff
     size_t consumedSoFar = 0;
-    while (consumedSoFar < strippedLength) {
+    while (consumedSoFar < strippedFields.size()) {
```

Comment:
Isn't `strippedFields` size known at compile time? Maybe the compiler is smart enough to make `strippedFields.size()` a constant, but I wonder if this could be a `sizeof()` or something?

Reply:
No need to try and optimize this. std::array::size() is constexpr (https://en.cppreference.com/w/cpp/container/array/size) so this does get resolved at compile time.